python/cugraph/cugraph/dask/link_prediction/jaccard.py

```diff
@@ -146,13 +146,16 @@ def jaccard(input_graph, vertex_pair=None, use_weight=False):
 
     if use_weight:
         raise ValueError(
-            "'use_weight' is currently not supported and must be set to 'False'"
+            "'use_weight' is currently not supported but will " "be in the next release"
```

Comment:
I would just say it's not supported and remove the "in the next release" part. I'm thinking schedule-related messaging should go in a blog or release note type of document.